.scripts/assets-manager/min.mjs

```diff
@@ -78,37 +78,30 @@ glob(config.source).then((files) => {
                     if (fileInfo.ext === ".js") {
                         let reader = await fs.readFile(file, "utf8");
```

Comment:
Here the issue I think is that fs on windows will read this file with `\r\n` and on linux it will read it with `\n`
Then when it creates the .map file it generates different mappings hash because that file length is longer on Windows making it fall on more lines making the hashes to differ.